pandas/tests/indexes/datetimes/test_arithmetic.py

```diff
@@ -41,6 +41,25 @@ def addend(request):
     return request.param
 
 
+class TestDatetimeIndexComparisons(object):
```

Comment:
can you lump the None with the NaT comparisons. Do we have the same testing of these comparisons for scalars?

Reply:
AFAICT the comparison tests are scattered.  I had planned to consolidate them here in a follow-up to keep narrow focus here.

> can you lump the None with the NaT comparisons

They would need to be separate tests since the behavior is different, but I can put the tests adjacent to each other in this class.

Reply:
Centralizing the DTI comparison tests is now a part of #19317.  After that I'll make a pass to make sure all the relevant cases are covered.